src/controllers/connection.controller.ts

```diff
@@ -3,13 +3,43 @@ import { RequestWithUser } from '@/interfaces';
 import { ConnectionService } from '@/services';
 
 class ConnectionController {
-  private connectionSerivce = new ConnectionService();
+  private connectionService = new ConnectionService();
 
   public getUserConnections = async (req: RequestWithUser, res: Response, next: NextFunction) => {
     try {
       const id = Number(req.params.user_id);
-      const data = await this.connectionSerivce.getUserConnections(id);
-      res.status(200).json({ data });
+      const data = await this.connectionService.getUserConnections(id);
+      return res.status(200).json({ data });
+    } catch (error) {
+      next(error);
+    }
+  };
+  public sendConnectionRequest = async (req: RequestWithUser, res: Response, next: NextFunction) => {
+    try {
+      const senderId = req.user.id;
+      const receiverId = Number(req.params.user_id);
```

Comment:
for recieverId wouldn't you use the RequestWithUser. Also you don't have to parse if you use it this way, its been parsed from before

Reply:
We are using RequestWithUser, that's where we get senderId. Receiver id is grabbed from params.

Reply:
And params return a string, so that's what parsing is for.